### PR TITLE
Flake8 errors resolved

### PR DESCRIPTION
**Pull request by tusharmakkar08**
Hey @alecthomas 

Earlier `tox` wasn't running successfully due to `flake8` issues. Let me know if we can upgrade `.travis.yml` and add `tox` there instead of `nosetests`. 

Thanks

### CODE REVIEW

File: voluptuous/schema_builder.py [hunk 1]
```diff
@@ -216,15 +221,11 @@ def _compile_mapping(self, schema, invalid_msg=None):
         invalid_msg = invalid_msg or 'mapping value'
 
         # Keys that may be required
-        all_required_keys = set(key for key in schema
-                                if key is not Extra
-                                and ((self.required and not isinstance(key, (Optional, Remove)))
-                                     or isinstance(key, Required)))
+        all_required_keys = set(key for key in schema if key is not Extra and (
+            (self.required and not isinstance(key, (Optional, Remove))) or isinstance(key, Required)))
```
Comment (alecthomas) [inline voluptuous/schema_builder.py]: I think this (and the below) are less clear. The underlying issue is just about operators starting a line, which can be simply fixed by putting just the operators at the end of the preceding line.

Comment (tusharmakkar08) [inline voluptuous/schema_builder.py]: Done.

**Comment by tusharmakkar08** [comment]
@alecthomas : Any comments?

**Comment by alecthomas** [comment]
👍

**Pull request merged**